.tests/integration/config.yaml

```diff
@@ -25,8 +28,26 @@ reference:
 
 trimmer_software: "fastp_pe"
 
-bedtools_intersect:
-  extra: "-header"
+arriba:
+  assembly: "reference/assembly.fasta"
+  blacklist: "reference/arriba_blacklist.txt"
+  container: "docker://hydragenetics/arriba:2.2.1"
+  extra: "-p abc.gff3 -k abc.tsv.gz"
+  gtf: "reference/assembly.gtf"
+
+arriba_draw_fusion:
+  container: "docker://hydragenetics/arriba:2.3.0"
+  cytobands: "/path/to/file.tsv"
+  gtf: "/path/to/file/Homo_sapiens.GRCh38.93.chr.gtf"
+  protein_domains: "/path/to/file.gff3"
+
+bcftools_SNV:
+  all: "/path/to/all.bed"
+  aml: "/path/to/aml.bed"
+
+bcftools_SV:
+  all: "/path/to/all.bed"
+  aml: "/path/to/aml.bed"
```

Comment:
This will not pass a .test/integration dry-run as these files does not exist!?
should be an empty file in .tests/integration/reference/all.bed and so on.

Reply:
weird, it did dry run without issues. but I'll fix those non-existent paths now